### ISSUE TITLE
Value assertion does not work on HTML <select> elements

### ISSUE THREAD

**Issue by pahund** [producthunt/chai-enzyme#80]
If I have this element:
```html
<input value={1} />
```
This assertion is true, as expected:
```javascript
element.should.have.value(1)
```
If I have this element:
```html
<select value={1}>
    <option value={1}>1</option>
</select>
```
The same assertion fails, value is supposedly undefined.

**Comment by ljharb** [comment]
@pahund because `select` doesn't have values, and its value depends on having a `selected` `<option>`. Try adding `selected` to the option tag, and remove "value" from `<select>`.

**Comment by pahund** [comment]
That's what I did at first, but when I do, React gives me a warning message on the console that I should use "value" on my select tag instead of "selected" on an option tag.

**Comment by pahund** [comment]
![eingang_-_pahund_-_private_nachrichten_-_motor-talk_de](https://cloud.githubusercontent.com/assets/1639313/15641705/ce161fa6-2642-11e6-9c28-5be19894e53d.png)

**Comment by ljharb** [comment]
Ah, interesting. In that case I think your expectation (that it works) is reasonable.

**Comment by ayrton** [comment]
Not too long ago I've added `value` support for `textarea` HTML elements in #63, I had already created an issue #64 to fix this for `select` HTML elements too. 

I will close #64 since this issue has more info. I can work on a solution when I have some spare time.

**Comment by pahund** [comment]
Thanks @ayrton and @ljharb, keep up the good work!

**Comment by ayrton** [comment]
Fixed in #82
